index.html

```diff
@@ -0,0 +1,190 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Static Web Quiz</title>
+  <link rel="stylesheet" type="text/css" href="quiz.css">
+</head>
+<body>
+
+<!-- CARD ONE -->
+<article>
+  <header>
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans|Share+Tech" rel="stylesheet">
```

Comment:
Make sure you put link tags like this in the `head` of your HTML. Preferably before your own quiz.css `link`